controllers/userController.js

```diff
@@ -82,18 +85,19 @@ exports.changePassword = async (req, res, next) => {
 
 exports.forgotPassword = async (req, res, next) => {
   try {
-    const validationErrors = userValidate.validateForgotPasswordCredentials(req.body);
-    if (validationErrors.length > 0) {
+    
+    const { email } = req.body;
+    if (!email) {
       return res.status(422).json({ errors: validationErrors });
     }
-    const { email } = req.body;
     const user = await userService.findUser({email});
     if (!user) {
       return res.status(404).json({ message: "User not found" });
     }
     const resetToken = await createToken.createToken(email, user._id)
     const expiration = Date.now() + 3600000;
     await userService.updateResetToken(user, resetToken, expiration);
```

Comment:
When creating services for specific models, it's advisable to design queries in a versatile manner that can serve multiple purposes. For instance, if you're developing a service for updating the user model and using a query like findOneAndUpdate, it's beneficial to make this query more general and give it a generic name. This approach allows you to utilize the same query for various purposes related to updating the user model, eliminating the need for multiple distinct services